options/posix/generic/arpa-inet-stubs.cpp

```diff
@@ -146,8 +146,32 @@ const char *inet_ntop(int af, const void *__restrict src, char *__restrict dst,
 	errno = ENOSPC;
 	return NULL;
 }
-int inet_pton(int, const char *__restrict, void *__restrict) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+int inet_pton(int af, const char *__restrict src, void *__restrict dst) {
```

Comment:
Should `dst` be zero'd before writing to it?

Reply:
Not that I know of, and musl doesn't seem to do it either unless I'm misreading

Reply:
Alright.